### PR TITLE
Find uml diagram

### PR DESCRIPTION
**Pull request by viki0526**

### CODE REVIEW

File: docs/diagrams/FindClassDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Class "FindCommand" as FindCommand
+Command <|-- FindCommand
+FindCommand --righto "1" Predicate : contains
+FindPersonDescriptor .up. FindCommand
+
+Name "*" *-up- FindPersonDescriptor
+Phone "*" *-up- FindPersonDescriptor
+Address "*" *-up- FindPersonDescriptor
+Tag "*" *-up- FindPersonDescriptor
```
Comment (ongkimlai) [inline docs/diagrams/FindClassDiagram.puml]: Should the composition be on the other side instead? The descriptor is the "whole" in this whole-part relationship.

File: docs/diagrams/FindOrSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+deactivate AddressBookParser
+
+activate FindOrPredicateParser
+FindCommandParser -> FindOrPredicateParser : parse()
+deactivate FindCommandParser
+
+activate FindCommand
+FindOrPredicateParser -> FindCommand
+
+@enduml
```
Comment (ongkimlai) [inline docs/diagrams/FindOrSequenceDiagram.puml]: You might have missed out exporting the image for this.

File: docs/diagrams/FindClassDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+Class "FindCommand" as FindCommand
+Command <|-- FindCommand
+FindCommand --righto "1" Predicate : contains
```
Comment (ongkimlai) [inline docs/diagrams/FindClassDiagram.puml]: Should the aggregation be on the other side instead? FindCommand contains a Predicate.

**Review by ongkimlai — DISMISSED** [review]
Some suggestions for changes

**Review by manu2002g — APPROVED** [review]
Lgtm!

**Review by ongkimlai — APPROVED** [review]
LGTM!

**Pull request merged**